packages/itwinui-css/src/backdrop/backdrop.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-backdrop {
+  z-index: 998;
+  isolation: isolate;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, var(--iui-opacity-4));
```

Comment:
We don't have variable for this one?

Reply:
what do you mean? A combined variable for `rgba(0, 0, 0, var(--iui-opacity-4))`? I don't see any